Update readmes (#59)
Sindri-Labs/sindri-resources: circuit_tutorials/noir/README.md

# Noir Circuit Tutorials

> In this directory, we provide a handful of Noir circuits tutorials which have been prepared to be used with Sindri.

Reviewer: ```suggestion
In this directory, we provide a handful of Noir circuit tutorials which have been prepared for use with Sindri.
```
